mne_bids/utils.py

```diff
@@ -26,9 +27,166 @@
 from mne.io.constants import FIFF
 from mne.time_frequency import psd_array_welch
 
+from mne_bids.config import BIDS_PATH_ENTITIES
 from mne_bids.tsv_handler import _to_tsv, _tsv_to_str
 
 
+class BIDSPath(object):
```

Comment:
have you updated `api.rst` ?

Reply:
Should this be added to `Utils`, or create a separate section for `Classes`?

Reply:
utils sounds reasonable to me.